Arista show version not working - template issue

##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 - Template Issue with error and raw data


##### TEMPLATE USING
<!----->
https://github.com/networktocode/ntc-templates/blob/master/ntc_templates/templates/arista_eos_show_version.textfsm
```
Value MODEL (\S+)
Value HW_VERSION (\S+)
Value SERIAL_NUMBER (\S+)
Value SYS_MAC (\S+)
Value IMAGE (\S+)
Value TOTAL_MEMORY (\d+)
Value FREE_MEMORY (\d+)
Value UPTIME (.+)

Start
  ^Arista\s+${MODEL}
  ^Hardware\s+version:(\s+${HW_VERSION})?
  ^Hardware\s+MAC\s+address:\s+\S+
  ^Serial\s+number:(\s+${SERIAL_NUMBER})?
  ^System\s+MAC\s+address:\s+${SYS_MAC}
  ^Software\s+image\s+version:\s+${IMAGE}
  ^Architecture:\s+\S+
  ^Internal\s+build\s+(version|ID):\s+\S+
  ^Image\s+(format\s+version|optimization):\s+\S+
  ^Uptime:\s+${UPTIME}
  ^Total\s+memory:\s+${TOTAL_MEMORY}
  ^Free\s+memory:\s+${FREE_MEMORY} -> Record
  ^. -> Error
```

##### SAMPLE COMMAND OUTPUT
<!----->

Physical device:
```
Arista DCS-7280SR-48C6-F
Hardware version: 12.00
Serial number: JPE20395540
Hardware MAC address: d4af.f7b0.8fb5
System MAC address: d4af.f7b0.8fb5

Software image version: 4.31.6M
Architecture: i686
Internal build version: 4.31.6M-39956901.4316M
Internal build ID: cdb8a50a-dbee-4c32-9ae6-33ec718ffcce
Image format version: 3.0
Image optimization: Sand-4GB

Uptime: 3 weeks, 2 days, 13 hours and 59 minutes
Total memory: 8051592 kB
Free memory: 5340020 kB
```
Virtual device:
```
Arista cEOSLab
Hardware version:
Serial number: 35BA12C16E793CAD48DE0EEC072CFD74
Hardware MAC address: 001c.73cd.2352
System MAC address: 001c.73cd.2352

Software image version: 4.31.6M-39953990.4316M (engineering build)
Architecture: x86_64
Internal build version: 4.31.6M-39953990.4316M
Internal build ID: 27a18095-d8a6-4dcf-a74c-c9ad0f655faf
Image format version: 1.0
Image optimization: None

cEOS tools version: (unknown)
Kernel version: 6.8.0-60-generic

Uptime: 2 days, 15 hours and 56 minutes
Total memory: 43138472 kB
Free memory: 27548760 kB
```




##### SUMMARY
<!--- Explain the problem briefly -->
Device Onboarding -> Sync Devices from network gives errors

Physical device:
```
2025-06-05 13:46:30.106225 | load_manufacturers | Error | — | 192.168.233.2: Unable to load Manufacturer due to a missing key in returned data, ('manufacturer',)
-- | -- | -- | -- | --
2025-06-05 13:46:30.108019 | load_platforms | Error | — | 192.168.233.2: Unable to load Platform due to a missing key in returned data, ('platform',)
2025-06-05 13:46:30.109573 | load_device_types | Error | — | 192.168.233.2: Unable to load DeviceType due to a missing key in returned data, ('device_type',)
2025-06-05 13:46:30.110771 | load_devices | Error | — | 192.168.233.2: Unable to load Device due to a missing key in returned data, ('device_type',), 'device_type'
2025-06-05 13:46:30.111819 | load_devices | Error | — | Unable to onboard 192.168.233.2, returned data missing for ['device_type', 'hostname', 'mgmt_interface', 'mask_length', 'serial', 'platform']
```


Virtual device: (cEOS)
```
2025-06-05 13:44:23.297024 | main | Debug | — | Schema validation failed for 172.20.20.8. Error: [] is not of type 'string', 'integer'Failed validating 'type' in schema['properties']['serial']: {'type': ['string', 'integer'], 'description': 'Serial number of the network device', 'minItems': 1}On instance['serial']: [].
-- | -- | -- | -- | --
2025-06-05 13:44:23.339241 | execute_command_getter | Debug | — | Command Getter Result: {'172.20.20.8': {'failed': True, 'failed_reason': 'Schema validation failed.'}}
2025-06-05 13:44:23.359144 | execute_command_getter | Debug | — | CommandGetter data type check result: True
2025-06-05 13:44:23.376365 | load_manufacturers | Debug | — | loading manufacturer data for 172.20.20.8
2025-06-05 13:44:23.392500 | load_manufacturers | Error | — | 172.20.20.8: Unable to load Manufacturer due to a missing key in returned data, ('manufacturer',)
2025-06-05 13:44:23.408387 | load_platforms | Debug | — | loading platform data for 172.20.20.8
2025-06-05 13:44:23.425988 | load_platforms | Error | — | 172.20.20.8: Unable to load Platform due to a missing key in returned data, ('platform',)
2025-06-05 13:44:23.442145 | load_device_types | Debug | — | loading device_type data for 172.20.20.8
2025-06-05 13:44:23.459342 | load_device_types | Error | — | 172.20.20.8: Unable to load DeviceType due to a missing key in returned data, ('device_type',)
2025-06-05 13:44:23.475391 | load_devices | Debug | — | loading device data for 172.20.20.8
2025-06-05 13:44:23.491215 | load_devices | Error | — | 172.20.20.8: Unable to load Device due to a missing key in returned data, ('device_type',), 'device_type'
2025-06-05 13:44:23.506349 | load_devices | Error | — | Unable to onboard 172.20.20.8, returned data missing for ['device_type', 'hostname', 'mgmt_interface', 'mask_length', 'serial', 'platform']
```


I've ran [Runs Commands on a Device to simulate SSoT Command Getter.]

Physical:
```
"show version": [{"model": "DCS-7280SR-48C6-F", "hw_version": "12.00", "serial_number": "JPE20395540", "sys_mac": "d4af.f7b0.8fb5", "image": "4.31.6M", "total_memory": "8051592", "free_memory": "5337728", "uptime": "3 weeks, 2 days, 14 hours and 30 minutes"}]
```

Virtual:
```
show version comes back like this: "show version": []
```

Ran this unit test:
https://github.com/networktocode/ntc-templates/blob/master/tests/arista_eos/show_version/arista_eos_show_version3.raw

Physical:
```
All the tests passed...
```

Virtual:
```
E       textfsm.parser.TextFSMError: State Error raised. Rule Line: 23. Input Line: cEOS tools version: (unknown)
```

##### STEPS TO REPRODUCE
<!----->
```
Device Onboarding -> Sync Devices
Define all the fields + Platform Arista
When do perform a dry run you will get above errors and nothing gets added
```

<!--- Paste example playbooks or commands between quotes below -->


<!--- You can also paste gist.github.com links for larger files -->

##### EXPECTED RESULTS
<!--- What did you expect to happen when running the steps above? -->
Device gets added to Nautobot - show version working

--- a/ntc_templates/templates/arista_eos_show_version.textfsm
+++ b/ntc_templates/templates/arista_eos_show_version.textfsm
@@ -20,4 +20,6 @@ Start
   ^Uptime:\s+${UPTIME}
   ^Total\s+memory:\s+${TOTAL_MEMORY}
   ^Free\s+memory:\s+${FREE_MEMORY} -> Record
+  ^cEOS\s+tools\s+version:
+  ^Kernel\s+version:
   ^. -> Error

--- a/tests/arista_eos/show_version/arista_eos_show_version_ceos.raw
+++ b/tests/arista_eos/show_version/arista_eos_show_version_ceos.raw
@@ -1,0 +1,19 @@
+Arista cEOSLab
+Hardware version:
+Serial number: 35BA12C16E793CAD48DE0EEC072CFD74
+Hardware MAC address: 001c.73cd.2352
+System MAC address: 001c.73cd.2352
+
+Software image version: 4.31.6M-39953990.4316M (engineering build)
+Architecture: x86_64
+Internal build version: 4.31.6M-39953990.4316M
+Internal build ID: 27a18095-d8a6-4dcf-a74c-c9ad0f655faf
+Image format version: 1.0
+Image optimization: None
+
+cEOS tools version: (unknown)
+Kernel version: 6.8.0-60-generic
+
+Uptime: 2 days, 15 hours and 56 minutes
+Total memory: 43138472 kB
+Free memory: 27548760 kB

--- a/tests/arista_eos/show_version/arista_eos_show_version_ceos.yml
+++ b/tests/arista_eos/show_version/arista_eos_show_version_ceos.yml
@@ -1,0 +1,10 @@
+---
+parsed_sample:
+  - free_memory: "27548760"
+    hw_version: ""
+    image: "4.31.6M-39953990.4316M"
+    model: "cEOSLab"
+    serial_number: "35BA12C16E793CAD48DE0EEC072CFD74"
+    sys_mac: "001c.73cd.2352"
+    total_memory: "43138472"
+    uptime: "2 days, 15 hours and 56 minutes"

